tests/unit/chambers_of_deputies/test_chamber_of_deputies_dataset.py

```diff
@@ -67,6 +68,15 @@ def test_clean_2017_reimbursements(self):
             with self.subTest():
                 assert(subquota in all_subquotas)
 
+    def test_translate_csv_with_reimbursement_with_net_value_with_comma(self):
+        csv_with_comma = os.path.join(self.fixtures_path, 'Ano-with-comma.csv')
+        with open(os.path.join(self.fixtures_path, 'reimbursements-with-decimal-point'), 'r') as csv_expected:
+            expected = csv_expected.read()
+        
+        xz_output = Dataset('')._translate_file(csv_with_comma)
+        output = lzma.open(xz_output).read().decode('utf-8')
```

Comment:
We need a context manager here too (`with lzma.open(…)`)… or at least to close the lzma file handler after reading it.

Reply:
I agree with that. I'm working on it.